wordpress-mobile/WordPress-iOS
# Fixes a couple of issues flagged by the static analyzer.

While investigating #12671 I ran the static analyzer to see if it could help illuminate the underlying problem. The analyzer flagged a number of issues, mostly around `nil` values. This PR is just a quick fix for a couple of the issues found.

<img width="507" alt="Screen Shot 2019-10-23 at 3 23 56 PM" src="https://user-images.githubusercontent.com/1435271/67431598-3c836400-f5aa-11e9-9d46-002c7d80bf7f.png">

To test:
Switch to `develop`
Run the static analyzer and confirm the issues. 
Switch to this branch
Confirm the two issues are resolved.

Hiya @shiki! Could I trouble you with this one? 

Update release notes:

- [x I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.
